packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1657,7 +1657,7 @@ class TransformLayer extends OffsetLayer {
       return null;
     final Vector4 vector = Vector4(localPosition.dx, localPosition.dy, 0.0, 1.0);
```

Comment:
Any reason we can't just use `MatrixUtils.transformPoint` here instead of re-implementing the transform logic here?

Reply:
It was like this before me, but the method looks valid. Fixed.